### ISSUE TITLE
Support colored output

### ISSUE THREAD

**Issue by prcastro** [tonsky/Clojure-Sublimed#99]
I expected the System.out console to support colors, but apparently it does not. The result is that my test suite and my code spills out colored debug messages, but I cannot see them properly on Sublime Text. Worse than that: the output has the color codes and it makes the output to format weirdly.

![Screenshot 2023-07-05 at 15 55 16](https://github.com/tonsky/Clojure-Sublimed/assets/1280742/7a7ea456-866c-4e1e-a2d1-5ac5146f5b4d)

Why don't we have a separate panel with the REPL, as we have with other editors/plugins, and keep the inline results for when it helps? It would solve a lot of the current problems (including what was mentioned in #94). Also, the blog post mentioned that an advantage is "there’s no extra panel to manage", but actually there is: today I already have to manage Sublime Console to check the output of my files, and sometimes it even prints unrelated stuff (like debug messages from plugins)!

In other words: as a user I have a lot of headaches because of Sublime Console, and no benefits compared to other solutions.

**Comment by tonsky** [comment]
Well, if I implemented REPL console myself, it would support even less features than whatever terminal you are using now. Plus, right now you have a choice of terminal. Have you trying using one that uses colors?

**Comment by prcastro** [comment]
What do you mean choice of terminal? I just start a nREPL on a separate terminal, but all the rest is done entirely on Sublime (inline results + stdout on console without colors). I don't see anything being output on my terminal window.

**Comment by tonsky** [comment]
That’s raw nREPL, right? It has the worst support. Other REPLs do not capture output, so you can see your stdout in the original terminal where you started the repl. That’s the mode I use and the mode this was designed around

**Comment by tonsky** [comment]
Ok, colors should now work for Raw nREPL:

![Screenshot 2024-03-12 at 15 29 13](https://github.com/tonsky/Clojure-Sublimed/assets/285292/4305f850-8c0a-4dc2-aa18-48b12dedb160)
